android/build.gradle

```diff
@@ -12,25 +12,34 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
+ * 
+ * ** Notice of Modification **
+ *
+ * This file has been altered from its original version by the Cryptomator team.
+ * For a detailed history of modifications, please refer to the version control log.
+ *
+ * The original file can be found at https://github.com/pCloud/pcloud-sdk-java
+ *
+ * --
+ *
+ * https://cryptomator.org/
  */
 
-apply from: file('../dependencies.gradle')
-
-def globalConfiguration = rootProject.extensions.getByName("ext")
 
 buildscript {
     repositories {
         mavenCentral()
         google()
         maven { url "https://plugins.gradle.org/m2" }
     }
-	dependencies {
-		classpath 'com.android.tools.build:gradle:7.2.2'
-	}
+    dependencies {
+        classpath dep.androidPlugin
+    }
 }
 
+apply from: rootProject.file('publishing.gradle')
+
 ext {
-    description = "The official Android SDK for integration with pCloud's API."
```

Comment:
Should we add here our own description?

Reply:
That's already handled:
https://github.com/cryptomator/pcloud-sdk-java/blob/a8a8a3587b70bde5c7a14f13b588711fa3ddd66d/maven-publish.gradle#L10